package.json

```diff
@@ -118,18 +120,13 @@
     }
   },
   "remarkConfig": {
+    "presets": [
+      "remark-preset-lint-recommended",
+      "remark-preset-lint-consistent"
```

Comment:
@jeddy3 Oh, and if you want you can also drop `remark-preset-` from both lines here.